pep-0690.rst

```diff
@@ -479,11 +523,26 @@ enabled.
 C API
 -----
 
-For authors of C extension modules, the proposed
-``importlib.set_lazy_imports()`` function will also be exposed in the stable C
-API as ``PyImport_SetLazyImports(PyObject *names_or_callback_or_null)``, and
-``importlib.is_lazy_import`` will be available as ``PyDict_IsLazyImport(PyObject
-*dict, PyObject *key)``.
+For authors of C extension modules, the proposed public C API is as follows:
```

Comment:
This might read much better as a table.

Reply:
I'm trying a table for this.

Reply:
Cool.  Ping me when you have something working, or feel like it's a dead end.

Reply:
I added a small table here.